### PR TITLE
Forcing output for create-sp command to json

### PR DESCRIPTION
**Pull request by jpoon**
I encountered a JSON decoding error when trying to parse the `az ad sp` command. This PR forces the output of the command to json. 

The exact error for those curious:

```
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 2 column 3 (char 4)
```

### CODE REVIEW

File: examples/generateManifest.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def main():
     print("Creating Service Principal")
     app_info = json.loads(
         subprocess.check_output(
-            "az ad sp create-for-rbac --role=Contributor --scopes /subscriptions/" + subscription_id + "/",
+            "az ad sp create-for-rbac --role=Contributor --scopes /subscriptions/" + subscription_id + "/ -o json",
```
Comment (itowlson) [inline examples/generateManifest.py]: I think we should also do this for the `az group create` command on line 49.

Comment (jpoon) [reply examples/generateManifest.py]: Fair point. Updated PR.

**Comment by itowlson** [comment]
LGTM

**Comment by jpoon** [comment]
ready to merge i think? cc @rbitia

**Comment by jpoon** [comment]
Looks like a similar PR was merged - https://github.com/Azure/aci-connector-k8s/commit/67af818f51b0e60c2b138b4a59b37b858ca46e60 so this is no longer necessary.

**Pull request closed without merging**